lib/hexpm_web/templates/layout/footer.html.eex

```diff
@@ -67,6 +67,7 @@
 
 
   <script src="<%= Routes.static_path(Endpoint, "/assets/app.js") %>"></script>
+  <script src="https://unpkg.com/@themesberg/flowbite@latest/dist/flowbite.bundle.js"></script>
```

Comment:
What is this used for?

Reply:
For the dropdown component, I opted to use https://flowbite.com/docs/getting-started/quickstart/ tailwindcss library. I used the CDN option to add it to the project